frontend/lib/src/theme/primitives/iconSizes.ts

```diff
@@ -18,6 +18,7 @@ export const iconSizes = {
   xs: "0.5rem",
   sm: "0.75rem",
   md: "0.9rem",
+  twoMd: "1rem",
```

Comment:
nit: could maybe call this one `reg` since its `1rem` or `base`?

Reply:
Really battling with this... probably makes more sense to be something like `base`, but then its inconsistent with our existing pattern 😭

Reply:
`base` won the internal struggle - updated 👍🏼